integration/test_collection.py

```diff
@@ -605,20 +606,82 @@ def test_search_hybrid_only_vector(collection_factory: CollectionFactory) -> Non
     assert len(objs) == 2
 
 
-@pytest.mark.parametrize("limit", [1, 5])
-def test_search_limit(collection_factory: CollectionFactory, limit: int) -> None:
+@pytest.mark.parametrize("limit", [1, 2])
+def test_hybrid_limit(collection_factory: CollectionFactory, limit: int) -> None:
     collection = collection_factory(
-        properties=[Property(name="Name", data_type=DataType.TEXT)],
+        properties=[Property(name="Name", data_type=DataType.TEXT, tokenization=Tokenization.WORD)],
         vectorizer_config=Configure.Vectorizer.none(),
     )
-    for i in range(5):
-        collection.data.insert({"Name": str(i)})
 
-    assert len(collection.query.fetch_objects(limit=limit).objects) == limit
+    res = collection.data.insert_many(
+        [
+            {"Name": "test"},
+            {"Name": "another"},
+            {"Name": "test"},
+        ]
+    )
+    assert res.has_errors is False
+    assert len(collection.query.hybrid(query="test", alpha=0, limit=limit).objects) == limit
+
+
+@pytest.mark.parametrize("offset,expected", [(0, 2), (1, 1), (2, 0)])
+def test_hybrid_offset(collection_factory: CollectionFactory, offset: int, expected: int) -> None:
```

Comment:
maybe move all of these to test_offset? but only if you want to

Reply:
I'd prefer them separate to decouple the different collection configs involved in each one